python/nvtx/nvtx.py

```diff
@@ -231,3 +233,28 @@ def end_range(range_id):
         The `RangeId` object returned by the `start_range` function.
     """
     libnvtx_end_range(range_id)
+
+
+def enabled():
+    """
+    Returns True if nvtx is enabled.
+    """
+    return not os.getenv("NVTX_DISABLE", False)
+
+
+if not enabled():
+    class annotate(contextlib.nullcontext):
+        def __init__(self, *args, **kwargs):
+            pass
+        def __call__(self, func):
+            return func
+
+    # Could use a decorator here but overheads are significant enough
+    # not to. See https://github.com/NVIDIA/NVTX/pull/24 for discussion.
+    _default = object()
```

Comment:
What's the reason for using `_default` instead of `None`?

Reply:
`None` I guess -- I figured there was a reason it was being done in the S/O answer above but `None` is as lightweight as `object()`. Fixed.